### PR TITLE
feat: Client now support vllm and image input (improved)

### PR DESCRIPTION
**Pull request by yongzhemiaolegemi**
+ typos fixed
+ useless "print" deleted

### CODE REVIEW

**Review by zmsn-2077 — APPROVED** [review]
LGTM

File: align_anything/serve/chatbot.py [hunk 1]
```diff
@@ -82,6 +84,8 @@ class SpecialCommand(Enum):
     EXIT = '/exit: End the dialogue and quit.'
     REGENERATE = '/regenerate: Regenerate the last response.'
     HELP = '/help: Show this help message.'
+    IMAGE = '/image: Select an image as input.'
+    VLLM = '/vllm: Whether to use VLLM model'
```
Comment (Gaiejj) [inline align_anything/serve/chatbot.py:88]: VLLM is a fast and easy-to-use library for LLM inference and serving while VLM is the Vision Language Model. Please rename this config.

File: align_anything/serve/chatbot.py [hunk 2]
```diff
@@ -174,11 +178,12 @@ def quit(self) -> NoReturn:
 class ModelArgs:
     model_name_or_path: str | os.PathLike
     temperature: float = 1.0
-    max_length: int = 512
+    max_length: int = 4096
     top_p: float = 1.0
     repetition_penalty: float = 1.0
     dtype: torch.dtype | str | None = 'auto'
     template: str = "Dialogue"
+    vllm: str = "False"
```
Comment (Gaiejj) [inline align_anything/serve/chatbot.py:186]: Same as above.

File: align_anything/serve/chatbot.py [hunk 3]
```diff
@@ -188,27 +193,35 @@ def __init__(
...
+        vllm: str = "False",
     ) -> None:
         """Initialize the chatbot."""
         
         self.name = os.path.basename(os.path.normpath(model_name_or_path))
         self.template = get_template_class(template)
+        if vllm =="True":
+            self.vllm=True
+        else:
+            self.vllm=False
```
[the remark this answers is not in the document]
Comment (yongzhemiaolegemi) [reply align_anything/serve/chatbot.py]: all fixed

**Review by Gaiejj — APPROVED** [review]
LGTM.

**Pull request merged**